code/modules/research/designs/mechfabricator_designs.dm

```diff
@@ -270,7 +270,7 @@
 	id = "gygax_armor"
 	build_type = MECHFAB
 	build_path = /obj/item/mecha_parts/part/gygax_armor
-	materials = list(MAT_METAL=25000,MAT_DIAMOND=10000)
+	materials = list(MAT_METAL=10000,MAT_DIAMOND=10000,MAT_TITANIUM=15000)
```

Comment:
I guess I should have clarified I meant replacing a costly material like uranium or diamond not raising costs. RD has suffered enough

Or maybe better yet just make this a silver/metal alloy instead of shoehorning a new material into mining when our economic system is already meaningless.